### PR TITLE
Task DEV-1

### PR DESCRIPTION
**Pull request by RomanMiranovich**

### CODE REVIEW

**Comment by AlexanderSoroka** [comment]
Please, use meaningful names for commits and pull requests rather than auto-generated by github\any other tools

File: Dev1.txt [hunk 1]
```diff
@@ -0,0 +1,13 @@
+class Dev1
```
Comment (AlexanderSoroka) [inline Dev1.txt]: codestyle violation

File: Dev1.txt [hunk 2]
```diff
@@ -0,0 +1,13 @@
+class Dev1
+{
+    public static void main(String[] args) {
+        for (int i = 0; i <= 100; i++) {
+            if (i % 3 == 0) {
+                int n = i / 3;
+                System.out.println(3 * n);
```
Comment (AlexanderSoroka) [inline Dev1.txt]: Task specification violation

**Review by AlexanderSoroka — CHANGES_REQUESTED** [review]

File: OutputOfNumbers.txt [hunk 3]
```diff
@@ -1,13 +1,13 @@
-class Dev1
+class OutputOfNumbers
```
Comment (AlexanderSoroka) [inline OutputOfNumbers.txt]: Any java code must be in *.java file, not txt

File: OutputOfNumbers.txt [hunk 4]
```diff
@@ -1,13 +1,13 @@
-class Dev1
+class OutputOfNumbers
 {
```
Comment (AlexanderSoroka) [inline OutputOfNumbers.txt]: codestyle violation (braces)

**Review by AlexanderSoroka — CHANGES_REQUESTED** [review]

**Comment by RomanMiranovich** [comment]
But in document with our tasks was written: "Java - код в блокноте"

2017-06-20 17:46 GMT+02:00 AlexanderSoroka <notifications@github.com>:

**Review by AlexanderSoroka — APPROVED** [review]

**Comment by AlexanderSoroka** [comment]
You wrote "But in document with our tasks was written: "Java - код в блокноте"" using notepad doesn't mean you must save everything only as txt file.

**Pull request merged**